website/api/controllers/get-human-interpretation-from-osquery-sql.js

```diff
@@ -49,19 +49,17 @@ Remember to minimize the number of words used!
 
 Please give me all of the above in JSON, with this data shape:
 
-\`\`\`
 {
   risks: 'TODO',
   whatWillProbablyHappenDuringMaintenance: 'TODO'
-}
-\`\`\``;
+}`;
```

Comment:
When I was testing this by sending gibberish strings, I noticed that the result of this was often adding additional text around the report. e.g., 

> Since the provided query appears to be nonsensical and not a valid osquery, it's impossible to anticipate any specific risks or maintenance outcomes from a laptop failing this policy. That being said, a generic response could be as follows:
> 
> ```json
> {
>   "risks": "Unpredictable behavior due to invalid osquery policy",
>   "whatWillProbablyHappenDuringMaintenance": "Maintenance may fail or result in unexpected outcomes due to invalid osquery policy"
> }
> ```

I tested adding another line to the prompt to make sure only the report was returned in the message, and it reduced the number of errors I saw from `JSON.parse()`. It can probably be tweaked more to use fewer prompt tokens, but here is what I added:

```suggestion
}

Please do not add any text outside of the JSON report or wrap it in a code fence.`;
```

Reply:
Nice!